py-polars/tests/unit/functions/range/test_date_range.py

```diff
@@ -138,15 +138,6 @@ def test_date_ranges_single_row_lazy_7110() -> None:
     assert_frame_equal(result, expected)
 
 
-def test_date_range_end_of_month_5441() -> None:
```

Comment:
Maybe this test can be repurposed to show the new behavior, instead of flat-out deleting it?